resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadLibrary.js

```diff
@@ -125,40 +125,13 @@
 	$.loadFacebookAPI = function (callback) {
 		if (window.wgEnableFacebookClientExt) {
 			return loadFacebookV2(callback);
```

Comment:
I think we should remove the ```if``` statement here.  The ```if``` was added to toggle between two versions of the FB library, not to decide whether to call FB code at all. Put another way, if this code looked like this:
```php
if (window.wgEnableFacebookClientExt) {
    return loadFacebookV2(callback);
} else if (window.wgEnableFBConnect) {
    return loadFacebookV1(callback);
}
```
then that would be a clear indication that this function is intended to be called in situations where no FB of any kind is loaded.  However, as it was written, some kind of FB code is always loaded when this function runs.